server/src/main/java/org/apache/cassandra/sidecar/job/OperationalJobManager.java

```diff
@@ -76,22 +81,33 @@ public OperationalJob getJobIfExists(UUID jobId)
      * The job execution failure behavior is tracked within the {@link OperationalJob}.
      *
      * @param job OperationalJob instance to submit
-     * @return OperationalJob instance that is submitted
      * @throws OperationalJobConflictException when the same operational job is already running on Cassandra
      */
-    public OperationalJob trySubmitJob(OperationalJob job) throws OperationalJobConflictException
+    public void trySubmitJob(OperationalJob job) throws OperationalJobConflictException
     {
         checkConflict(job);
 
         // New job is submitted for all cases when we do not have a corresponding downstream job
-        return jobTracker.computeIfAbsent(job.jobId, jobId -> job);
+        jobTracker.computeIfAbsent(job.jobId, jobId -> {
+            internalExecutorPool.executeBlocking(() -> {
+                job.execute(Promise.promise());
+                return null;
+            });
+            return job;
+        });
```

Comment:
Although `executeBlocking` with promise is deprecated, let's still use it for the succinct code it produces here. We can migrate away from the deprecated method all together in future refactoring. 

```suggestion
        jobTracker.computeIfAbsent(job.jobId, jobId -> {
            internalExecutorPool.executeBlocking(job::execute);
            return job;
        });
```